jekyll/_cci2/circleci-images.md

```diff
@@ -91,11 +92,12 @@ version: 2.0
 jobs:
   build:
     docker:
-      - image: circleci/ruby:2.4.2-jessie-node
+      - image: cimg/ruby:2.7.1-node
     steps:
       - checkout
       - run:
           name: "Update Node.js and npm"
+          # what should I do
```

Comment:
I assume this is coming out?

Reply:
Good find. Silly me thinking that's a comment. I this a good config example to keep here? Is suggesting running an npm upgrade valuable?